packages/contracts/contracts/optimistic-ethereum/OVM/bridge/messaging/OVM_L2CrossDomainMessenger.sol

```diff
@@ -185,7 +185,6 @@ contract OVM_L2CrossDomainMessenger is
      * @return _valid Whether or not the message is valid.
      */
     function _verifyXDomainMessage()
-        view
```

Comment:
Could you explain why this is no longer a `view`? If I understand correctly, solidity will now `CALL` instead of `STATICCALL`

Reply:
My mistake. I removed this line because of the way I was previously getting `OVM_L1MessageSender` to work until I figured out that I could do the `return` thing in the constructor.

Reply:
Slight nit that this doesn't need a diff

Reply:
Whoops

Reply:
I'm gonna keep the diff because I don't feel like running the integration tests again if I don't have to :-/

Reply:
We need faster CI :(